embedded-graphics/src/geometry/point.rs

```diff
@@ -78,11 +78,66 @@ impl Point {
         Point { x, y }
     }
 
+    /// Creates a point with X and Y values set to an equal value.
+    ///
+    /// ```rust
+    /// use embedded_graphics::geometry::Point;
+    ///
+    /// let filled = Point::new_equal(11);
+    ///
+    /// assert_eq!(filled, Point { x: 11, y: 11 });
+    /// ```
+    pub const fn new_equal(fill: i32) -> Self {
+        Point { x: fill, y: fill }
+    }
+
```

Comment:
```suggestion
    /// Creates a point with X and Y values set to an equal value.
    ///
    /// ```rust
    /// use embedded_graphics::geometry::Point;
    ///
    /// let point = Point::new_equal(11);
    ///
    /// assert_eq!(point, Point { x: 11, y: 11 });
    /// ```
    pub const fn new_equal(value: i32) -> Self {
        Point { x: value, y: value }
    }

```